plugin/SpreadsheetPage.qml

```diff
@@ -21,80 +21,80 @@ import Sailfish.Silica 1.0
 import org.kde.calligra 1.0 as Calligra
 
 DocumentPage {
-    id: page;
+    id: page
 
     attachedPage: Component {
         SpreadsheetListPage {
-            document: doc;
+            document: doc
         }
     }
 
     Calligra.Document {
-        id: doc;
+        id: doc
     }
 
     Calligra.View {
-        id: v;
+        id: v
```

Comment:
Agreed and was about to change while at it. Didn't instantly figure out a better so didn't do that yet. Could continue later with more cleanups.